### PR TITLE
refactor(treasury): Convert RESM to NESM

### PR DESCRIPTION
**Pull request by kriskowal**
Refs #527

### CODE REVIEW

**Comment by katelynsills** [comment]
I think @Chris-Hibbert is better suited for this review.

File: packages/treasury/test/test-bootstrapPayment.js [hunk 1]
```diff
@@ -28,7 +34,7 @@ const makeInstall = async (root, zoe) => {
 
 test('bootstrap payment', async t => {
   const zoe = makeZoe(fakeVatAdmin);
-  const autoswapInstall = await makeInstall(autoswapRoot, zoe);
+  const autoswapInstall = await makeInstall(await autoswapRoot, zoe);
```
Comment (Chris-Hibbert) [inline packages/treasury/test/test-bootstrapPayment.js]: This is  a non-top-level await. Is that going to be allowed? Can we put the await inside `makeInstall()` instead?

File: packages/treasury/test/test-vault.js [hunk 2]
```diff
@@ -46,7 +46,8 @@ trace('makeZoe');
  * @param {string} sourceRoot
  */
 async function launch(zoeP, sourceRoot) {
-  const contractBundle = await bundleSource(require.resolve(sourceRoot));
+  const contractPath = new URL(await importMetaResolve(sourceRoot, import.meta.url)).pathname;
```
Comment (Chris-Hibbert) [inline packages/treasury/test/test-vault.js]: await is not top-level here.

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]

**Review by Chris-Hibbert — APPROVED** [review]
LGTM!

**Pull request merged**